readme.md

```diff
@@ -522,3 +522,4 @@ Some users may want to extend Zustand's feature set which can be done using thir
 ## Comparison with other libraries
 
 - [Difference between zustand and valtio](https://github.com/pmndrs/zustand/wiki/Difference-between-zustand-and-valtio)
+- [NPM package downloads trend of zustand](https://npm-compare.com/zustand/#timeRange=THREE_YEARS)
```

Comment:
Hmm, it's no longer a comparison.

On second thought, I don't feel like adding something about NPM downloads in readme.md.
Please consider adding it in https://docs.pmnd.rs/zustand/getting-started/comparison, and then _probably_ comparing with other libraries that are listed in the doc makes sense.